en/docs/install-and-setup/setup/advance-configurations/configure-certificate-revocation.md

```diff
@@ -0,0 +1,43 @@
+# Verifying Certificate Revocation
+
+The default HTTPS transport listener (Secured Pass-Through) can verify with the certificate authority whether a certificate is still trusted before it completes an SSL connection. If the certificate authority has revoked the certificate, a connection will not be completed.
+
+When this feature is enabled, the transport listener verifies client
+certificates when a client tries to make an HTTPS connection with the server. Therefore, the client needs to send it's public certificate along with the requests to the server.
+
+When this feature is enabled, the server attempts to
```

Comment:
Consecutive paragraphs are starting with the same phrase. Better if we can combine the 2 paragraphs together or change the starting phrase.